docs/av-init.1.md

```diff
@@ -0,0 +1,10 @@
+# av-init 1 "" av-cli "Aviator CLI User Manual"
+
+# NAME
+
+av-init - Initialize the repository for Aviator CLI.
+
+# DESCRIPTION
+
+Aviator CLI internally stores metadata in the repository. This command
+initializes the metadata storage for the repository.
```

Comment:
should we specify that they need to create a gh access token when initiatilizing?